app/components/Documentation/components/ViewContent.tsx

```diff
@@ -16,33 +31,157 @@ const styled = {
 type ViewContentProps = {
   documentationArticle: DocumentationArticle;
   isEditDocumentation?: boolean;
+  documentationMenu?: DocumentationItemMenuType;
+  locale: string;
 };
 
 const ViewContent: React.FC<ViewContentProps> = ({
   documentationArticle,
   isEditDocumentation,
-}): JSX.Element => (
-  <>
-    <Wrapper
-      className="mb-to-sm-0 mb-from-sm-3"
-      css={{
-        ...(isEditDocumentation && styled),
-      }}
-    >
-      <H3>
-        <span className="d-none d-md-inline-block">
-          {documentationArticle?.title}
-        </span>
-      </H3>
-    </Wrapper>
-    <Wrapper
-      css={{
-        ...(isEditDocumentation && styled),
-      }}
-    >
-      <TextBlock content={documentationArticle?.content} />
-    </Wrapper>
-  </>
-);
+  documentationMenu,
+  locale,
+}): JSX.Element => {
+  const treeArray = getFlatDataFromTree({
+    treeData: documentationMenu?.map((node) => ({ ...node })),
+    getKey: (node) => node.id,
+    getNodeKey: ({ treeIndex }) => treeIndex,
+    ignoreCollapsed: false,
+  });
+  const currentArticleId = documentationArticle?.id;
+  const currentArrayIndex = treeArray.filter(
+    (item) => item?.node.id == currentArticleId,
+  )[0]?.treeIndex;
+  const sliceTitle = (title: string): string =>
+    title?.length < 26 ? title : title?.substr(0, 25) + '...';
+  const isStartArticle = currentArrayIndex < 1;
+  const isLastArticle = currentArrayIndex == treeArray.length - 1;
+
+  const onClickPrevArticle = (): void => {
```

Comment:
you used the same functions in another component, this is a duplication, find a way to merge

Reply:
done: https://github.com/peeranha/peeranha-web/pull/1255/commits/c018592c251a49f7d793813c5d02d03715e687c9